### ISSUE TITLE
Compile failure with libc 0.2.90

### ISSUE THREAD

**Issue by bgilbert** [tailhook/openat#35]
https://github.com/rust-lang/libc/pull/2116 made `libc::RENAME_EXCHANGE` unsigned:

```
error[E0308]: mismatched types
   --> src/dir.rs:371:13
    |
371 |             libc::RENAME_EXCHANGE)
    |             ^^^^^^^^^^^^^^^^^^^^^ expected `i32`, found `u32`
    |
help: you can convert a `u32` to an `i32` and panic if the converted value doesn't fit
    |
371 |             libc::RENAME_EXCHANGE.try_into().unwrap())
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by cgwalters** [comment]
PR in https://github.com/tailhook/openat/pull/36

**Comment by tailhook** [comment]
Released as v0.1.20
